src/sql/azdata.d.ts

```diff
@@ -2336,6 +2341,7 @@ declare module 'azdata' {
 	 * Represents a provider of resource
 	 */
 	export interface ResourceProvider {
+		providerLanguageId: string;
```

Comment:
it seems unusual to have the language id communicated in this interface.  The other methods have to do with creating firewall rules.  Is this change related to firewall rules using Kusto?